yangvalidator/v2/views.py

```diff
@@ -388,31 +465,45 @@ def create_output(
         if xym_response is None:
             response_args = {
                 'data': {
-                    'Error': 'Failed to load any yang modules. Please provide at least one'
+                    'Message': 'Failed to load any yang modules. Please provide at least one'
                     ' yang module. File must have .yang extension',
+                    'Type': 'error',
                 },
                 'status': 400,
             }
         elif xym_response.get('stderr'):
             response_args = {
-                'data': {'Error': f'Failed to fetch content of {url}', 'xym': xym_response},
+                'data': {
+                    'Message': f'Failed to fetch content of {url}',
+                    'xym': xym_response,
+                    'Type': 'error',
+                },
                 'status': 404,
             }
         else:
-            short_url = os.path.basename(str(url)).removesuffix('.txt')
+            short_url = os.path.basename(str(url))[:-4]  # name of draft without .txt
             response_args = {
-                'data': {'Error': f'No modules were extracted using xym from {short_url}', 'xym': xym_response},
+                'data': {
+                    'Message': f'No modules were extracted using xym from {short_url}',
+                    'xym': xym_response,
+                    'Type': 'info',
+                },
             }
         response = JsonResponse(**response_args)
     elif choose_options:
         existing_dependencies, found_repo_modules = checker.get_existing_dependencies()
-        json_body['dependencies'] = {'missing': missing, 'existing': existing_dependencies}
+        json_body['dependencies'] = {
+            'missing': missing,
+            'existing': existing_dependencies,
+        }
         if len(missing) == 0 and not found_repo_modules:
             response = validate(request, xym_response, json_body)
         else:
             response = JsonResponse({'output': json_body}, status=202)
     elif latest:
-        json_body['dependencies'] = {'repo-modules': checker.get_latest_revision()}
+        json_body['dependencies'] = {
```

Comment:
Was this change (and other similar ones above) made automatically by black? If so, I think you might have it misconfigured locally. The initialization should stay on one line if it fits into the max line length, see [here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines). The trailing comma is what prevented black from trying to collapse it back in github actions.

Reply:
yeah, I think it was automatic, I will check what is wrong with pre-commit

Reply:
i will make the changes while refactoring repo for this issue
https://github.com/YangCatalog/yang-validator-extractor/issues/109